pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -235,6 +237,12 @@ def update_time_status_change(self, person: Person, time: float):
         # Assigns the time of status change using current time and transition
         # time:
         if transition_time < 0:
+            logging.info(f"Person: {person}")
+            logging.info(f"Status: {person.infection_status}")
+            logging.info(f"Next status: {person.next_infection_status}")
+            logging.info(f"transition_time: {transition_time}")
+            logging.info(f"Key worker: {person.key_worker}")
+            logging.info(f"Care home resident: {person.care_home_resident}")
```

Comment:
I know you did this for debugging purposes, but I wonder whether this is too much information, even for the lowest logging level (info). Happy to discuss if you want to commit this though

Reply:
Whoops, didn't realise that I had committed this, sorry. I'll remove these lines now and look through other files to make sure that other ones that shouldn't be there are removed.